src/common/store/synchronization.slice.ts

```diff
@@ -15,17 +15,24 @@ import {
 import type { HandshakeKeyPair, ReduxIPC } from '@types';
 import { safeJSONParse } from '@utils';
 
+export enum SynchronizationTarget {
+  RENDERER = 'RENDERER',
+  MAIN = 'MAIN',
+  SIGNING = 'SIGNING'
+}
```

Comment:
Maybe name this `Process` instead of `SynchronizationTarget`. Also, maybe we should follow [TypeScript´s naming conventions for enums](https://www.typescriptlang.org/docs/handbook/enums.html), e.g., `Renderer`, `Main`, `Signing`.